### PR TITLE
Cu 864e851w1 areas table common viktormarinho

### PR DESCRIPTION
**Pull request by viktormarinho**
##### Description of changes

- [x] Why (Why did you make this pull request): Create Separate table for sectors in auth & update layout & types where needed
- [x] Brief description of the changes: 
- [x] Possible breakouts (Possible places where things could break):

### CODE REVIEW

**Review by vitormiura — APPROVED** [review]
gtg, just need to resolve conflicts before merging 👍

File: apps/auth/src/areas.ts [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    const areas = await prisma.area.findMany();
+
+    return res.status(200).json({ areas });
+})
+
+const updateAreaSchema = z.object({
+    sectorId: z.string()
+});
+
+router.patch("/update-area", authenticate, async (req, res) => {
```
Comment (abehidek) [inline apps/auth/src/areas.ts:19]: I think changing this PATCH endpoint route from PATCH /update-area to just PATCH /area is more concise and it's good to follow the REST standards

File: apps/auth/src/areas.ts [hunk 2]
```diff
@@ -0,0 +1,49 @@
+import { Router } from "express";
+import { z } from "zod";
+import { prisma, Prisma } from "./lib/prisma";
+import { authenticate, authorizeAdmin } from "./lib/middleware";
+import type { Request, Response } from "express";
+
+const router = Router();
+
+router.get('/areas', authenticate, async (_req, res) => {
```
Comment (abehidek) [inline apps/auth/src/areas.ts:9]: By following the same standards of the comment above, GET /areas -> GET /area

File: apps/auth/src/__tests__/auth-router.spec.ts
Comment (abehidek) [inline apps/auth/src/__tests__/auth-router.spec.ts:1]: Why did this test was commented on

**Comment by abehidek** [comment]
Nonetheless great PR

**Review by abehidek — APPROVED** [review]

**Pull request merged**